datafusion/core/tests/sql/explain_analyze.rs

```diff
@@ -90,7 +90,7 @@ async fn explain_analyze_baseline_metrics() {
     assert_metrics!(
         &formatted,
         "CoalescePartitionsExec",
-        "metrics=[output_rows=3, elapsed_compute="
+        "metrics=[output_rows=5, elapsed_compute="
```

Comment:
this is due to the plan changing in the same way as explained in `datafusion/core/tests/sql/window.rs`